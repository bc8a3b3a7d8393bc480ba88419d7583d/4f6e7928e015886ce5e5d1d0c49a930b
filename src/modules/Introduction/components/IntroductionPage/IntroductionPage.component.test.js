import { testSnapshots } from '../../../../utils/test.util';

import Component from './IntroductionPage.component';

describe('Modules/Landing - IntroductionPage', () => {
  testSnapshots(Component, [
    {
      description: 'basic render',
      props: {
        data: {
          title: 'title',
          text: 'text',
          image: 'page1',
        },
      },
    },
  ]);
});
