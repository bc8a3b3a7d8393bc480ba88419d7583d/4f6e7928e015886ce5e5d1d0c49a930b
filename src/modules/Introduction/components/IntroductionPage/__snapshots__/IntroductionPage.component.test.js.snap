// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modules/Landing - IntroductionPage snapshots basic render 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "paddingHorizontal": 20,
      "paddingVertical": 50,
    }
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "testUri": "../../../src/assets/images/onboarding/page1.png",
      }
    }
    style={
      Object {
        "height": 217,
        "width": 282,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "fontFamily": "Oswald-Regular",
        "fontSize": 16,
        "marginBottom": 20,
        "marginTop": 10,
        "textAlign": "center",
      }
    }
  >
    title
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "fontFamily": "Oswald-Regular",
        "fontSize": 12,
        "marginBottom": 20,
        "textAlign": "center",
      }
    }
  >
    text
  </Text>
</Component>
`;
